packages/css-engine/src/core/to-value.ts

```diff
@@ -39,5 +44,26 @@ export const toValue = (
       fallbacks.length > 0 ? `, ${fallbacks.join(", ")}` : "";
     return `var(--${value.value}${fallbacksString})`;
   }
-  return value.value;
+
+  if (value.type === "keyword") {
+    return value.value;
+  }
+
+  if (value.type === "invalid") {
+    return value.value;
+  }
+
+  if (value.type === "unset") {
+    return value.value;
+  }
+
+  if (value.type === "rgb") {
+    return `rgba(${value.r}, ${value.g}, ${value.b}, ${value.alpha})`;
```

Comment:
with your alpha as a value from 0-100 this wasn't supposed to work as you need to add % to it, but I would keep it this way and instead use numbers from 0 to 1, because they don't require %, wdyt?